src/site/stages/build/process-cms-exports/transformers/node-person_profile.js

```diff
@@ -0,0 +1,38 @@
+const { getDrupalValue } = require('./helpers');
+
+const transform = entity => ({
+  entityType: 'node',
+  entityBundle: 'person_profile',
+  entityUrl: {
+    path: entity.path[0].alias,
```

Comment:
Is this not using the helper you wrote a while back because it doesn't have the breadcrumbs?

Reply:
That was a different helper.  At some point we may also want a helper for this situation though.